ui/app/src/views/config/PluginsList.tsx

```diff
@@ -0,0 +1,118 @@
+// Copyright 2025 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { PluginModuleResource } from '@perses-dev/plugin-system';
+import { Box, Card, CardContent, Typography, Divider, Button, CircularProgress, Stack } from '@mui/material';
+import Grid from '@mui/material/Grid2';
+import { ReactElement, useEffect, useState } from 'react';
+import { useSnackbar } from '@perses-dev/components';
+import { PluginDetailsDialog } from './PluginDetailsDialog';
+
+export function PluginsList(): ReactElement {
+  const [plugins, setPlugins] = useState<PluginModuleResource[]>([]);
+  const [error, setError] = useState<string | null>(null);
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const [selectedPluginModule, setSelectedPluginModule] = useState<PluginModuleResource | null>(null);
+  const { exceptionSnackbar } = useSnackbar();
+
+  useEffect(() => {
+    const fetchPlugins = async (): Promise<void> => {
+      try {
+        const res = await fetch('/api/v1/plugins');
+        if (!res.ok) {
+          throw new Error(`Failed to fetch plugins: ${res.statusText}`);
+        }
+        const data = await res.json();
+        setPlugins(data);
+      } catch (err: unknown) {
+        if (err instanceof Error) {
+          setError(err.message);
+        } else {
+          setError('An unknown error occurred');
+        }
+      } finally {
+        setIsLoading(false);
+      }
+    };
+    fetchPlugins();
+  }, []);
+
+  const handleOpenPluginDetails = (pluginModule: PluginModuleResource): void => {
+    setSelectedPluginModule(pluginModule);
+  };
+
+  const handleClosePluginDetails = (): void => {
+    setSelectedPluginModule(null);
+  };
+
+  if (isLoading) {
+    return (
+      <Stack width="100%" sx={{ alignItems: 'center', justifyContent: 'center', height: '50vh' }}>
+        <CircularProgress />
+      </Stack>
+    );
+  }
+
+  if (error) {
+    exceptionSnackbar(error);
+  }
+
+  return (
+    <Box sx={{ p: 2 }}>
+      <Grid container spacing={3}>
+        {plugins.map((pluginModule) => (
+          <Grid size={{ xs: 12, sm: 6, md: 4, lg: 2 }} key={pluginModule.metadata.name}>
+            <Card elevation={2} sx={{ height: '100%' }}>
+              <CardContent>
+                <Typography variant="h3" component="div" gutterBottom>
```

Comment:
I don't see div utility here
```suggestion
                <Typography variant="h3" gutterBottom>
```

Reply:
fixed in b5aff23d5e30d8a817cf9b6c2abd6a706b7fb0a2